upup/models/cloudup/resources/addons/storage-aws.addons.k8s.io/v1.15.0.yaml.template

```diff
@@ -16,7 +15,7 @@ kind: StorageClass
 metadata:
   name: gp2
   annotations:
-    storageclass.kubernetes.io/is-default-class: "false"
+    storageclass.beta.kubernetes.io/is-default-class: "false"
```

Comment:
is this addition still needed? I see upstream docs refer to the non-beta annotation.

Reply:
This was a mistake. Reverted.